guides/inventory/inventory-units.md

```diff
@@ -0,0 +1,40 @@
+# Inventory units
+
+<!-- TODO:
+  This article is a stub.
+-->
+
+A `Spree::InventoryUnit` object is created every time that an item is sold.
+It tracks a sold item as it moves from an item in the warehouse to an item in
+a shipment.
+
+Note that the `Spree::InventoryUnit` tracks an item as an object associated with
+a `Spree::Order`, a `Spree::Shipment` and a specific `Spree::LineItem`. This
+allows you to more closely track the status of an order and the line items and
+shipments associated with it.
+
+A `Spree::InventoryUnit` object has the following attributes:
+
+- `state`: The current state of the inventory unit. The state value can be
+  `on_hand`, `backordered`, `shipped`, or `returned`.
+- `variant_id`: The ID for the `Spree::Variant` corresponding with the inventory
+  unit that has been sold.
+- `order_id`: The ID for the `Spree::Order` associated with the current
```

Comment:
I'm not sure if it's worth specifying that this attribute is no more [on the Inventory Unit but it's calculated from the associated shipment of the Inventory Unit](#2377).

Reply:
Thank you for pointing this out! For now, I will simply remove this item from the list.